src/EntityManager.d.ts

```diff
@@ -0,0 +1,64 @@
+import { ObjectPool } from "./ObjectPool.js";
+import QueryManager from "./QueryManager.js";
+import EventDispatcher from "./EventDispatcher.js";
+import { SystemStateComponent } from "./SystemStateComponent.js";
+
+export class EntityManager {
+
+  constructor(world);
+
+  getEntityByName(name);
+
+  /**
+   * Create a new entity
+   */
+  createEntity(name);
+
+  entityAddComponent(entity, Component, values);
```

Comment:
Don't these functions need return types?

Reply:
I will fix that after my holidays thanks!